ambientweather2sqlite/database.py

```
@@ -211,9 +214,26 @@ def query_daily_aggregated_data(
         }
 
     """
+    # Validate timezone if provided
+    if tz:
+        try:
+            ZoneInfo(tz)
+        except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)